### ISSUE TITLE
Robotic grasp detection using a novel two-stage approach. (arXiv:2011.14123v1 [cs.RO])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#1535]
Robotic grasp detection using a novel two-stage approach. (arXiv:2011.14123v1 [cs.RO])<br>
https://ift.tt/2Vl0PGJ<br>
<p>Recently, deep learning has been successfully applied to robotic grasp detection. Based on convolutional neural networks (CNNs), there have been lots of end-to-end detection approaches. But end-to-end approaches have strict requirements for the dataset used for training the neural network models and it's hard to achieve in practical use. Therefore, we proposed a two-stage approach using particle swarm optimizer (PSO) candidate estimator and CNN to detect the most likely grasp. Our approach achieved an accuracy of 92.8% on the Cornell Grasp Dataset, which leaped into the front ranks of the existing approaches and is able to run at real-time speeds. After a small change of the approach, we can predict multiple grasps per object in the meantime so that an object can be grasped in a variety of ways.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
